CONTRIBUTING.md

```diff
@@ -57,127 +59,61 @@ yarn watch
 
 ### Testing
 
-For all projects, run from project root. For single project, run from
-package root.
-
-All packages:
+For all packages, run from repository root. For single package, run from
+package folder.
 
 ```shell
-# from project root
+# Test once
 yarn test
 
 # Watch for changes and re-run
 yarn test -- --watch
 ```
 
-### Editor Setup
-
-#### VSCode
-
-We recommend using the ESLint plugin with the following settings:
-
-NOTE: We enable `formatOnSave` so that /examples projects are formatted automatically. eslint --fix
-would only apply to code under /packages
-
-```json
-  "editor.formatOnSave": true,
-  "eslint.enable": true
-```
-
 ### Sandbox
 
-The [sandbox](examples/sandbox/README.md) app can be used as a local development aid.
+The [sandbox app](examples/sandbox/README.md) is available to demo usage and act as a development aid.
 
-### Branching Strategy
+It is wired up to use the local source of the Search UI libraries as dependencies via Lerna.
+This means as you update library code you can see it updated live in the sandbox app.
+  
+To run it locally:
 
-Our `master` branch holds the latest development code for the next release. If the next release will be a minor release, the expecation is that no breaking changes will be in `master`. If a change would be breaking, we need to put it behind a feature flag, or make it an opt-in change. We will only merge breaking PRs when we are ready to start working on the next major.
-
-All PRs should be created from a fork, to keep a clean set of branches on `origin`.
-
-Releases should be performed directly in master (or a minor branch for patches), following the Publishing guide below.
-
-We use a `stable` branch to indicate the latest release code.
-
-We will create branches for all minor releases.
-
-### Publishing
```

Comment:
The publishing instructions moved to a separate PUBLISHING doc since they are only useful to the internal team